packages/devextreme/js/__internal/grids/new/grid_core/inferno_wrappers/tree_view.ts

```diff
@@ -4,7 +4,10 @@ import dxTreeView from '@js/ui/tree_view';
 
 import { InfernoWrapper } from './widget_wrapper';
 
-export class TreeView extends InfernoWrapper<TreeViewProperties, dxTreeView> {
+export class TreeView<TItem extends TreeViewItemProperties> extends InfernoWrapper<
```

Comment:
Minor

```suggestion
export class TreeView<TItem = TreeViewItemProperties extends TreeViewItemProperties > extends InfernoWrapper<
```

It's strange that you updated code here but we didn't need to update its usage